experimental/ixdtf/src/parser.rs

```diff
@@ -556,50 +543,54 @@ mod test {
     #[test]
     fn test_bad_date() {
         let dt = "-2022-06-05".as_bytes();
-        let parsed = DateTimeParser::parse(dt);
+        let parsed = DateTimeParser { bytes: dt }.parse();
```

Comment:
nit: we should probably have a constructor like `DateTimeParser::new()`

Reply:
sg!